erpnext/public/js/utils/serial_no_batch_selector.js

```diff
@@ -354,14 +354,11 @@ erpnext.SerialNoBatchSelector = Class.extend({
 								return;
 							}
 
-							let batch_number = me.item.batch_no ||
-								this.grid_row.on_grid_fields_dict.batch_no.get_value();
-
 							if (me.warehouse_details.name) {
 								frappe.call({
 									method: 'erpnext.stock.doctype.batch.batch.get_batch_qty',
 									args: {
-										batch_no: batch_number,
+										batch_no: val,
```

Comment:
please change the variable name. it should be self-descriptive.

Reply:
I've updated the variable at its definition itself.